plugins/outofbandmanagement-drivers/ipmitool/src/main/java/org/apache/cloudstack/outofbandmanagement/driver/ipmitool/IpmitoolOutOfBandManagementDriver.java

```diff
@@ -37,6 +37,7 @@
 import java.util.List;
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.Executors;
+import org.apache.commons.lang3.StringUtils;
```

Comment:
please use our own (cloudstack) Stringutils as proxy, even when calling the apache.commons

Reply:
There is no consensus on that. I would prefer this way, using the library directly. We do not get benefits on a facade like that.

Reply:
yes we do, and we are badly hampered in most cases where we don't (gson/logging mostly)

Reply:
So, lets add a facade to Spring as well, and to any other utils library that we use :)

I still see no benefits on creating such facades.

Reply:
that is a lazy exaggeration, we have a facade for string utils and we are hampered by several 3rd party libraries for which we don't. If there are resources for making such a thing for spring, great. Let's do it.

BTW, in this case I didn't :-1: on this issue, but on other counts I might as well. duckduck "using 3rd party library facades", will give you some wisdom on this.

out